src/app/api/tickets/route.ts

```diff
@@ -0,0 +1,37 @@
+import { NextRequest, NextResponse } from "next/server";
```

Comment:
According to this google doc here `https://docs.google.com/document/d/1ndsqBGtGYWV4gKC_vZasDCXSYchIdpztfJAysoz5B-w/edit?tab=t.0`, we're naming the endpoint `/api/resident-request/?` (this also matches the name of the table too so it makes sense instead of 'tickets'? Thoughts?